src/olympia/amo/tests/test_cron.py

```diff
@@ -55,6 +56,16 @@ def test_file_uploads_deletion_oserror(self, storage_mock):
         assert storage_mock.delete.call_args_list[0][0][0] == fu_older.path
         assert storage_mock.delete.call_args_list[1][0][0] == fu_old.path
 
+    def test_delete_fake_emails(self, storage_mock):
+        fe_old = FakeEmail.objects.create(message='This is the oldest fake email')
+        fe_old.update(created=self.days_ago(360))
+        fe_new = FakeEmail.objects.create(message='This is the newest fake email')
+        fe_new.update(created=self.days_ago(45))
+
+        gc()
+        # FakeEmail which are older than 90 were deleted.
+        assert FakeEmail.objects.count() == 1
```

Comment:
Suggestion:

```suggestion
        assert FakeEmail.objects.count() == 1
        assert FakeEmail.objects.filter(pk=fe_new.pk).count() == 1
```

That way we also prove we deleted the right one.